card/mod/05_standard/set/type/user.rb

```diff
@@ -4,25 +4,26 @@
 attr_accessor :email
 
 format :html do
-
-  view :setup, tags: :unknown_ok, perms: lambda { |r| Auth.needs_setup? } do |args|
+  view :setup,
+       tags: :unknown_ok, perms: lambda { |r| Auth.needs_setup? } do |args|
```

Comment:
Unused block argument - `r`. If it's necessary, use `_` or `_r` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.